### DISCUSSION TITLE
themeConfig.blog.links支持扩展嘛

### DISCUSSION THREAD

**Discussion by zhangchengk** [vuepress-theme-hope/vuepress-theme-hope#574] [Q&A] [answered]
themeConfig.blog.links 有什么方式扩展嘛？比如我想加一个CSDN的 :smile:

**Comment by Mister-Hope** [comment, marked as answer]
暂时没有，我可以在下一个版本给你加一个。

支持扩展倒是有考虑过，不过用户可能很难找到风格比较统一的图标，尝试过效果不太好最后就放弃了。

**Reply by Sciroccogti** [reply]
我也想加个 bilibili 来着~

是否考虑直接把图标改为 iconfont，而不是代码内置的 svg？这样感觉会方便很多，然后默认的图标就放在默认的 css 里就行了，这也是很多主题的方法。

**Reply by Mister-Hope** [reply]
很抱歉很晚回复，漏掉了。iconfont 已经在页面上多处用于其他用途，而且对于国外用户来说，体验真的不好 （iconfont只在国内访问速度比较良好）

v2 有计划重构这一部分实现按需加载。

**Reply by Mister-Hope** [reply]
用户将可以自行提供 SVG 图标。

**Reply by Sciroccogti** [reply]
> iconfont 已经在页面上多处用于其他用途，而且对于国外用户来说，体验真的不好 （iconfont只在国内访问速度比较良好）

或者可以用户自定义 css，用 css 增加 font。

**Reply by Sciroccogti** [reply]
> 用户将可以自行提供 SVG 图标。

能否和导航栏一样使用 FontClass 呢？
